src/main/python/pre_fx_data.py

```diff
@@ -12,11 +12,11 @@
 from ipfx.bin.run_qc import qc_summary
 from ipfx.stimulus import StimulusOntology, Stimulus
 from ipfx.data_set_utils import create_data_set
-
+import ipfx.sweep_props as sp
```

Comment:
Can you import the items you need from sweep props? The two-letter module imports can be annoying to read / search unless they are very well known (e.g. np).